reactor-core/src/main/java/reactor/core/publisher/FluxIterable.java

```diff
@@ -103,10 +101,10 @@ public Object scanUnsafe(Attr key) {
 	 * Common method to take an {@link Iterator} as a source of values.
 	 *
 	 * @param s the subscriber to feed this iterator to
-	 * @param it the {@link Iterator} to use as a predictable source of values
+	 * @param sp the {@link Iterator} to use as a predictable source of values
```

Comment:
Iterator -> Spliterator; also in other javadocs below

Reply:
fixed